dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/MpTelemetryLogMappingUtils.java

```diff
@@ -361,16 +364,23 @@ private static void mapAccessToOpenTelemetry(LogRecordBuilder builder, String ev
                 } else if (key.equals("datetime") || key.equals("accessLogDatetime")) {
                     builder.setTimestamp(formatDateTime((String) value));
                 } else if (key.contains("requestHeader") || key.contains("responseHeader")) {
-
-                    String[] headerSplit = Arrays.stream(((String) value).split(",")).map(String::trim).toArray(String[]::new);
-
-                    attributes.put(formattedKey, headerSplit);
+                    if (key.contains("traceparent"))
+                        customSpan = createSpan((String) value, 0);
```

Comment:
Similar as above comment, add these hardcoded names to the Constants class, to make it more cleaner.